### PR TITLE
Supplemental Claims | Add primary phone selection page

### PR DESCRIPTION
**Pull request by Mottie**
## Description

Add What is your primary phone number page. It is only visible if the Veteran has both a home and mobile number within their profile. If only one number is defined, this page is skipped.

The `va-radio` hint text is dependent on component-library v1.13.1, so an update was needed.

## Original issue(s)

[#46511](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46511)

## Code changes

- Added primary phone question custom page
  - Only visible when Veteran has both a home and mobile number
  - Skipped when the Veteran has either a home or mobile number defined
- Update submit transformer to use:
  - Selected primary phone, or
  - Either home or mobile, if only one is available

## Testing done

- Added unit tests

## Screenshots

What is your primary phone number selection page, showing a home and mobile number radio selection with the actual number below

<details><summary>Primary phone not selected error</summary>

No primary phone selection showing 'You need to choose a primary phone number' error message
</details>

## Acceptance criteria
- [x] Show primary phone selection page if home & mobile numbers available
- [x] Skip primary phone if only one number is defined
- [x] Add unit tests
- [x] Add e2e test

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/appeals/995/components/PrimaryPhone.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          onVaValueChange={handlers.onSelection}
+          required
+        >
+          <va-radio-option
+            id="home-phone"
+            label="Home phone number"
+            value="home"
+            name="primary"
+            checked={primary === 'home'}
+            data-number={getFormatedPhone(homePhone)}
```
Comment (Mottie) [inline src/applications/appeals/995/components/PrimaryPhone.jsx:81]: Plan to change this `data-number` to `description` once the [va-radio-option description addition PR is merged](https://github.com/department-of-veterans-affairs/component-library/pull/564)

**Review by rocio-desantiago — APPROVED** [review]
Paired on and saw that it works in live session.

**Review by it-harrison — APPROVED** [review]

File: src/applications/check-in/day-of/tests/e2e/translation/validate-veteran-page-is-translated.cypress.spec.js [hunk 2]
```diff
@@ -46,12 +46,11 @@ describe('Check In Experience - Tagalog', () => {
   it('Validate Veteran page - tagalog', () => {
     // App is translated.
     ValidateVeteran.validatePage.dayOf('tl');
-    // NOTE: DS components don't seem to support tagalog..
     cy.get('[data-testid="last-name-input"]')
       .shadow()
       .find('label')
       .should('be.visible')
-      .and('have.text', 'Ang iyong apelyido (*Required)');
+      .and('have.text', 'Ang iyong apelyido (*Kailangan)');
```
Comment (Mottie) [inline src/applications/check-in/day-of/tests/e2e/translation/validate-veteran-page-is-translated.cypress.spec.js:53]: @patient-check-in This PR has updated the component library, so this test needed updating to pass

Comment (acrollet) [reply src/applications/check-in/day-of/tests/e2e/translation/validate-veteran-page-is-translated.cypress.spec.js:53]: Thanks for the update!

**Review by acrollet — APPROVED** [review]
Approved for check-in change

**Pull request merged**